selenpy/element/base_element.py

```diff
@@ -64,6 +64,7 @@ def click(self):
         element.click()
         
     def send_keys(self, *value):
+        self.find_element().clear()
```

Comment:
Should not put clear here. If you want to clear before entering, use it on page methods

Reply:
I updated